blazor/breadcrumb/templates.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Breadcrumb Templates with Blazor Breadcrumb component | Syncfusion
-description: Breadcrumb section explains how to customize the item template and separator template to the Breadcrumb items.
+description: Checkout and learn here all about templates support in Blazor Breadcrumb component of Syncfusion and more.
```

Comment:
Change like this - Checkout and learn here all about templates support in Syncfusion Blazor Breadcrumb component, its elements, and more.

Reply:
changed